### PR TITLE
Add priorityClassName to filebeat chart

### PR DESCRIPTION
**Pull request by tanordheim**
This adds the support for `priorityClassName` on the filebeat chart, the same way that got added to the elasticsearch and kibana charts in #109. As in the other PR, it defaults to no `priorityClassName` to avoid breaking anything.

- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [x] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

File: filebeat/README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ helm install --name filebeat elastic/filebeat --set imageTag=7.3.0
...
 | `affinity`               | Configurable [affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)                                                                                                                                                      | `{}`                                                                                                                      |
+| `priorityClassName`      | The [name of the PriorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass). No default is supplied as the PriorityClass must be created first.                                                                               | ``                                                                                                                        |
```
Comment (Crazybus) [inline filebeat/README.md]: This should be an empty string not an empty value (doesn't really matter though).

```suggestion
| `priorityClassName`      | The [name of the PriorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass). No default is supplied as the PriorityClass must be created first.                                                                               | `""`                                                                                                                        |
```

**Comment by Crazybus** [comment]
jenkins test this please

File: filebeat/README.md (hunk 1 above)
Comment (tanordheim) [reply filebeat/README.md]: I just went with how this was configured in the other charts that also have `priorityClassName` available; I'm happy to change it if you wish?

Comment (Crazybus) [reply filebeat/README.md]: Go for it! If you could fix it in the other charts too that would be great :)

Comment (tanordheim) [reply filebeat/README.md]: I'll handle it in a separate PR.

**Pull request merged**